### PR TITLE
Added more utilities for the Bot API

### PR DESCRIPTION
**Pull request by nekofurball**
TL;DR, **added 2 more data responses**, changed bot tag color and gave a suggestion for helping people make responsive bots.

### Added
- message.god 
     Gives a true or false response if the user is god or not.
- message.bot
     Gives a true or false response if the user is a bot or not.

### Changed
- the bot tag color (the previous color was absolutely GOD DAMN atrocious)

### ~~Suggestion~~
- ~~bots can use HTML~~
     ~~This would help with embedding, but would also have a catch being exploitation.~~
     ~~I think you should add an XSS filter to only allow certain tags (eg. div, span, img)~~
Pushed an XSS filter for bots to the merge.

_Any questions? nekofurball#3635 is my Discord tag._

_EDIT:_ If you need a module to help you with an XSS filter, I personally recommend the module [xss](https://npmjs.com/package/xss) as it allows you to add custom filter rules. (edited 11/11/2018 3:43 PM AEST)

_EDIT 2_: Added a bot XSS filter to the fork, thus adding those commits to the merge.
(edited 12/11/2018 4:18 PM AEST)

### CODE REVIEW

**Comment by 1024x2** [comment]
> bots can use HTML

oof, implementing a proper xss filter for bots would be very hard

what about markdown instead?

**Comment by nekofurball** [comment]
As I mentioned in the edit, there's a module that allows XSS filtering called xss. It allows custom filters and is very versatile. I think that could help us here. I could also attempt an implementation of this myself.

**Comment by nekofurball** [comment]
image
The xss module successfully helped me create an XSS filter for bots.
Regular users still can't use HTML
image

I will push this version to the master branch of my fork in a moment, adding this to the list of commits.

**Comment by nekofurball** [comment]
Pushed the xss filter to the fork.

**Comment by 1024x2** [comment]
looks noice, but please try to add semicolons when making pull requests in the future
